source/_integrations/flick_electric.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Flick Electric
+description: Instructions on how to set up the Flick Electric Pricing sensor in Home Assistant.
+logo: flick_electric.png
+ha_category:
+  - Energy
+ha_release: '0.110'
+ha_iot_class: Cloud Polling
+ha_codeowners: 
+  - '@ZephireNZ'
+ha_domain: flick_electric
+---
+
+[Flick Electric Co](https://www.flickelectric.co.nz/) is a power company in New Zealand, based around a transparent pricing model where each component of pricing is provided. This integration uses the mobile app's API from Flick Electric Co to get the current power price as well as each of the components price.
+
+## Configuration
+
+This integration can be configured via the Home Assistant frontend.
+
+- Go to **Configuration** -> **Integrations**.
+- Click on the `+` in the bottom right corner to add a new integration.
+- Search and select the **Flick Electric** integration form the list.
+- Follow the instruction on screen to add the sensor.
+
+<div class='note'>
+The configuration uses the client ID and secret used by the app at time of release. If this stops working, you can find the new ones by using a MITM proxy with the mobile app. The app will call `https://api.flick.energy/identity/oauth/token` with the `client_id` and `client_secret`.
```

Comment:
The inline code in this does not show up correctly, but I'm using the same format as [the Sonos page](https://github.com/home-assistant/home-assistant.io/blob/25b3e7c7fb36d534ab0fb5d6b3e2450eef57d1de/source/_integrations/sonos.markdown#service-sonossnapshot) so not sure what I'm doing wrong.

Reply:
Add enters around markdown. The parser needs to be able to switch context between HTML & Markdown. A new line will do that.

Fixed it 😉